bigtable/client/testing/embedded_server_test_fixture.cc

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/testing/embedded_server_test_fixture.h"
+#include "bigtable/client/internal/throw_delegate.h"
+#include <thread>
+
+namespace bigtable {
+namespace testing {
+
+void EmbeddedServerTestFixture::StartServer() {
+  int port;
+  std::string server_address("[::]:0");
+  builder_.AddListeningPort(server_address, grpc::InsecureServerCredentials(),
+                            &port);
+  builder_.RegisterService(&bigtable_service_);
+  builder_.RegisterService(&admin_service_);
+  server_ = builder_.BuildAndStart();
+  address_ = "localhost:" + std::to_string(port);
+  is_server_started_ = true;
```

Comment:
Consider simply `server_started_`, or `server_thread_active_`.

Reply:
removed as it was not needed